client/src/components/Projects/ProjectNew/index.jsx

```diff
@@ -1,9 +1,114 @@
-import React from 'react'
-
-export default function ProjectsNew() {
-  return (
-    <div>
-      ProjectsNew
-    </div>
-  )
+import React, { Component } from "react";
+import "./style.css";
+import TableMember from "./TableMember/TableMember";
+
+export default class index extends Component {
+  state = {
+    selection: {
+      rowKey: null,
```

Comment:
I didn't understand why do you need the rowKey?
I don't see that you use it asside of the error checking